File: 릿코드/4-median-of-two-sorted-arrays/4-median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int[] list = Stream.of(nums1, nums2).flatMapToInt(Arrays::stream).sorted().toArray();
        int listLen = list.length, index = listLen/2;
        double result = list[index];
        if(listLen%2 ==0) result = (double)(list[index-1] + list[index])/2;
        return result;
    }
}

File: 릿코드/4-median-of-two-sorted-arrays/4-median-of-two-sorted-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
const findMedianSortedArrays = (nums1, nums2) => {
    const nums = [...nums1, ...nums2].sort((a, b) => a-b);
    const index = parseInt(nums.length/2);
    if(nums.length%2 == 0) return ((nums[index-1] + nums[index])/2).toFixed(5);
    return nums[index].toFixed(5);
};